catalog-info.yaml

```diff
@@ -555,25 +555,29 @@ spec:
           message: Daily trigger of Snapshot DRA Pipeline per branch
           env:
             PIPELINES_TO_TRIGGER: 'logstash-dra-snapshot-pipeline'
+            INCLUDE_NEW_BRANCHES: '7.17'
```

Comment:
This seems good plan to keep DRA but isn't early to remove 7.17 from https://storage.googleapis.com/artifacts-api/snapshots/branches.json (AFAIK 6-months after 9.0 release from support perspective)?

Reply:
[storage.googleapis.com/artifacts-api/snapshots/branches.json](https://storage.googleapis.com/artifacts-api/snapshots/branches.json) uses https://artifacts-api.elastic.co/v1/versions and that API relies on artifacts being generated at least once every 30 days or so. We are waiting for the official Release API that the platform release productivity team are working on.

To play safe, I thought it would be interesting to support `INCLUDE_NEW_BRANCHES` to explicitly say what other branches need to be supported. But we can skip this for now and revisit it if needed in the future

Reply:
Awesome! Thank you for the explanation and makes sense to me.